docs/static/monitoring/configuring-logstash.asciidoc

```diff
@@ -20,26 +20,32 @@ monitoring cluster will show the Logstash metrics under the _monitoring_ cluster
 
 . Verify that the `xpack.monitoring.collection.enabled` setting is `true` on the
 production cluster. If that setting is `false`, the collection of monitoring data
-is disabled in {es} and data is ignored from all other sources.
-
-. Configure your Logstash nodes to send metrics by setting the
-`xpack.monitoring.elasticsearch.url` in `logstash.yml`. If {security} is enabled,
-you also need to specify the credentials for the 
-{stack-ov}/built-in-users.html[built-in `logstash_system` user]. For more information about these settings, see <<monitoring-settings>>.
+is disabled in {es} and data is ignored from all other sources. 
+
+. Configure your Logstash nodes to send metrics. 
+Set `xpack.monitoring.enabled` to `true` and set the
+`xpack.monitoring.elasticsearch.url` in `logstash.yml`. If {security} is
+enabled, you also need to specify the credentials for the
+{stack-ov}/built-in-users.html[built-in `logstash_system` user]. For more
+information about these settings, see <<monitoring-settings>>.
 +
 --
 [source,yaml]
 --------------------------------------------------
-xpack.monitoring.elasticsearch.url: ["http://es-prod-node-1:9200", "http://es-prod-node-2:9200"] <1>
-xpack.monitoring.elasticsearch.username: "logstash_system" <2>
+xpack.monitoring.enabled: true <1>
+xpack.monitoring.elasticsearch.url: ["http://es-prod-node-1:9200", "http://es-prod-node-2:9200"] <2>
+xpack.monitoring.elasticsearch.username: "logstash_system" <3>
 xpack.monitoring.elasticsearch.password: "changeme"
 --------------------------------------------------
-<1> If SSL/TLS is enabled on the production cluster, you must
+<1> Turns monitoring on. Note that the
```

Comment:
To me, this doesn't really make clear that it's a two-part problem. The clarifying "note" tells me that Elasticsearch needs to be configured in a certain way, but by the time we're here my head is entirely in configure-logstash mode (which means I assume this is still a setting that goes in the `logstash.yml`).

> <1> Turns monitoring on in Logstash, enabling Logstash to send monitoring data to Elasticseach. Note that Elasticsearch will _ignore_ any monitoring events it receives unless it is separately configured with `xpack.monitoring.collection.enabled: true`.